src/pages/ibm-logos/8-bar.mdx

```diff
@@ -492,17 +492,17 @@ their solutions as enabled by IBM, associations and events IBM has sponsored
 around the world, or co-marketing requests. Each request is evaluated
 individually, with the key criteria being appropriateness, clarity of
 relationship, clarity of ownership of communications or offering, general
-liability and so on. IBM requires all divisions to comply with these guidelines
-to ensure a consistent IBM brand identity across all messaging.
+liability and so on. IBM requires all third parties to comply with these guidelines to ensure a consistent IBM brand identity across all messaging.
 
 <Row className="resource-card-group">
 <Column colMd={4} colLg={4} noGutterSm>
     <ResourceCard
-      subTitle={<>Third-party IBM logo request<br />(IBM ID required)</>}
-      href="https://prdpcrhibmbl01.w3-969.ibm.com/marketing/logotool/logotool.nsf/BrandingLogoHome?Openform"
+      actionIcon="download"
+      subTitle={"IBM logo and brand guidelines for third-party users"}
+      href="https://www.ibm.com/design/language/files/IBM_Logo_3rdParties_300822.pdf"
```

Comment:
@Danilo-Colonheze-IBM Is this URL correct? Either I don't have access, or the URL is incorrect

<img width="1109" alt="image" src="https://user-images.githubusercontent.com/3360588/190654933-11923a1e-79fb-48fe-afd4-f79b6bb05829.png">

Reply:
Ah I just noticed this is because the url is hardcoded to use the root domain. I think if this is made relative it should work in both the deploy preview as well as production?

```suggestion
      href="/files/IBM_Logo_3rdParties_300822.pdf"
```

Reply:
Thank you for the review and suggestion @tay1orjones
I've chosen the absolute path based on previous experiences and other instances of files here in IDL.
So, happy to try and learn this way too